There is no gi-docgen on system. Therefore we need to -Dgtk_doc=false.

Solaris specific. Not suitable for upstream.

--- evince-45.0/meson.build
+++ evince-45.0/meson.build
@@ -165,11 +165,6 @@ libxml_dep = dependency('libxml-2.0', ve
 
 m_dep = cc.find_library('m')
 
-gidocgen_dep = dependency('gi-docgen', version: '>= 2021.1',
-			  fallback: ['gi-docgen', 'dummy_dep'],
-			  required: get_option('gtk_doc'),
-			  native: true)
-
 # Although GTK+ 3.10 includes hi-dpi functionality, it does not require a cairo with
 # cairo_surface_set_device_scale(), which we also need if we're to support hi-dpi,
 # so we need check for that explicitly.
